optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/statistic/StatsUtil.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2015 JBoss Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.benchmark.impl.statistic;
+
+import java.util.List;
+
+import org.optaplanner.benchmark.impl.result.SolverProblemBenchmarkResult;
+import org.optaplanner.core.api.score.Score;
+import org.optaplanner.core.impl.score.ScoreUtils;
+
+public class StatsUtil {
```

Comment:
Mind renaming this to StatisticUtils?
- All other Util(s) are plural: Utils not Util
- Don't abbreviate if it's not too verbose.